mne/io/meas_info.py

```diff
@@ -830,10 +830,17 @@ def _repr_html_(self, caption=None):
         meas_date = self['meas_date']
         if meas_date is not None:
             meas_date = meas_date.strftime("%B %d, %Y  %H:%M:%S") + ' GMT'
+        projs = self['projs']
+        if projs:
+            projs = ', '.join(
```

Comment:
Based on [the output](https://28584-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/30_info.html) it seems like this might be better:
```suggestion
            projs = '\n'.join(
```